js/main.js

```diff
@@ -0,0 +1,141 @@
+const $ul = document.querySelector('ul');
+const $searchButton = document.querySelector('.search-button');
+const $query = document.querySelector('#query');
+const jikan = 'https://api.jikan.moe/v4/anime?q=';
+const $form = document.querySelector('.form');
+
+$searchButton.addEventListener('click', searchFor);
+
+function searchFor(event) {
+  event.preventDefault();
+  const url = jikan + $query.value;
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', url);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', () => {
+    // console.log(xhr.response);
+    for (let i = 0; i < xhr.response.data.length; i++) {
+      const $li = document.createElement('li');
+      $li.setAttribute('class', 'column-third');
+
+      const $img = document.createElement('img');
+      $img.setAttribute('src', xhr.response.data[i].images.webp.image_url);
+      $li.appendChild($img);
+
+      const $p = document.createElement('p');
+      $p.setAttribute('class', 'title');
+      if (xhr.response.data[i].title_english !== null) {
+        $p.textContent = xhr.response.data[i].title_english;
+      } else {
+        $p.textContent = xhr.response.data[i].title_japanese;
+      }
+      $li.appendChild($p);
```

Comment:
I would break this up into another function like what we did with `renderEntry` in Code Journal